### DISCUSSION TITLE
How to set the octave of a chord ?

### DISCUSSION THREAD

**Discussion by JossPabloPC** [melanchall/drywetmidi#259] [Q&A] [answered]
I want to to add some chords to a patter builder but i want them to be an octave lower. Apparently when you create a chord only the names of the notes are used, I cant find a way to do this in the documentation.

**Comment by melanchall** [comment, marked as answer]
Hi,

You can add chords via `PatternBuilder` in many different ways. Please see small example:

```csharp
var patternBuilder = new PatternBuilder()

    // Add chord with two notes: A#3 and D4
    .Chord(new []
    {
        Note.Get(NoteName.ASharp, 3),
        Octave.Get(4).D
    })
    
    // Add chord with three notes: C#4 E4 G4
    .SetOctave(Octave.Get(4))
    .Chord(new Chord(
        NoteName.CSharp,
        NoteName.E,
        NoteName.G));
```

**Reply by melanchall** [reply]
@JossPabloPC Have I answered your question?

**Reply by JossPabloPC** [reply]
Yes, indeed. Thanks you so much !
